clusterloader2/cmd/clusterloader.go

```diff
@@ -100,6 +101,27 @@ func completeConfig(f *framework.Framework) error {
 	return nil
 }
 
+func printClusterNodes(f *framework.Framework) error {
+	nodes, err := util.GetSchedulableUntainedNodes(f.GetClientSet())
```

Comment:
Let's list all the nodes (not just schedulable and untainted) - something that may initially be tainted may be untainted later e.g.

Reply:
Done.
However I also added information about node being schedulable/unschedulable to log. IMHO this information will be useful, due to the nodes parameter from cluster config being initialize based on this property.